core/src/main/scala/org/locationtech/rasterframes/util/DataFrameRenderers.scala

```diff
@@ -48,6 +48,12 @@ trait DataFrameRenderers {
               base64(rf_render_png(rf_resample(resolved, 0.5), ColorRamps.Viridis)), // TODO: how to expose?
               lit("\"></img>")
             )
+          else if (renderTiles && c.dataType == BinaryType)
+            when(
+              substring(resolved, 0, 8) === lit(Array[Byte](137.asInstanceOf[Byte], 80, 78, 71, 13, 10, 26, 10)),
```

Comment:
Consider making this a boolean UDF in the companion class called something like `is_png(data: Array[Byte]): Boolean`.

Reply:
So this guy is a Column function... So the signature would be like so


```
def is_png(col: TypedColumn[BinaryType, Any]): TypedColumn[BooleanType, Any]
```

IMO not sure if it's worth doing.